complex/ft_comp.c

```diff
@@ -54,8 +54,6 @@ static int ft_open_cntrs(void)
 {
 	int ret;
 
-	opts.comp_method = test_info.cq_wait_obj;
-
```

Comment:
Ok, so the original idea was that ft_cntr_open can open a counter by calling ft_cntr_set_wait_attr and switching on opts.comp_method so I was trying to set opts.comp_method before calling ft_cntr_open to open the counter with the appropriate wait_obj (and I incorrectly assumed the config file was using the same enum in the cq_wait_obj that opts.comp_method was using to set the wait_obj of cntr_attr). Does that make a bit more sense...?